tests/phpunit/CRM/Utils/UrlTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Psr\Http\Message\UriInterface;
+
+/**
+ * Class CRM_Utils_UrlTest
+ * @group headless
+ */
+class CRM_Utils_URLTest extends CiviUnitTestCase {
```

Comment:
Given the class being tested is named `CRM_Utils_Url` shouldn't the test be `CRM_Utils_UrlTest` instead of `CRM_Utils_URLTest`?

Reply:
opps - it should have been removed when I moved it to URLTest after moving the functions - gone now

Reply:
I'm not sure I'm following. I was talking about the class names. The class being tested is named `CRM_Utils_Url`, but the test class is `CRM_Utils_URLTest` (while the file is UrlTest.php). Basically, you're using all uppercase letters for the URL word in the test class name and `Url` for all the rest

Reply:
ah - good point!